docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -127,7 +126,7 @@ While previous versions of the cache configurations only logged warnings when th
 Due to that, you need to set up monitoring to disconnect the two sites in case of a site failure.
 The Keycloak High Availability Guide contains a blueprint on how to set this up.
 
-. While previous LoadBalancer configurations will continue to work with {project_name}, consider upgrading
+. While previous load balancer configurations will continue to work with {project_name}, consider upgrading
```

Comment:
This either needs to be either "upgrading an existing Route53 configuration to avoid..." or "upgrading existing Route53 configurations to avoid...".